src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -921,7 +921,9 @@ FROM
 	pg_dist_partition
 WHERE
 	logicalrelid = 'mx_test_schema_1.mx_table_1'::regclass
-	OR logicalrelid = 'mx_test_schema_2.mx_table_2'::regclass;
+	OR logicalrelid = 'mx_test_schema_2.mx_table_2'::regclass
+ORDER BY
+	logicalrelid;
```

Comment:
Should we "order by" by using ::text instead?

https://github.com/citusdata/citus/pull/6813#discussion_r1154207498

Reply:
what is the problem of ordering by the OID? Mostly curious, we can keep the text (though I vaguely remember text ordering is also broken sometimes for the tests if running different collations)